java-manta-client/pom.xml

```diff
@@ -170,6 +170,20 @@
             <version>${dependency.urlbuilder.version}</version>
         </dependency>
 
+        <!-- object cloning library for memoization of encryption state -->
+        <dependency>
+            <groupId>uk.com.robust-it</groupId>
```

Comment:
There are a few different deep cloneing libraries, any reason why this one in particular?

The readme says "Also cloning of files and streams might make the JVM crash." Are Files or Streams among the things we are cloning?